plugins/usbdmx/ShowJockeyDMXU1.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ShowJockeyDMXU1.h
+ * The synchronous and asynchronous ShowJockeyDMXU1 widgets.
+
+ * Adapted from SJ-DMX, p3root - Patrik Pfaffenbauer,
+ * patrik.pfaffenbauer@p3.co.at,
+ * https://github.com/p3root/SJ-DMX.git
+ *
+ * by Nicolas Bertrand, nbe@anomes.com
+ *
+ * Copyright (C) 2016 Nicolas Bertrand
+ */
+
+#ifndef PLUGINS_USBDMX_SHOWJOCKEYDMXU1_H_
+#define PLUGINS_USBDMX_SHOWJOCKEYDMXU1_H_
+
+#include <libusb.h>
+#include <memory>
+#include <string>
+#include "libs/usb/LibUsbAdaptor.h"
+#include "ola/DmxBuffer.h"
+#include "ola/base/Macro.h"
+#include "ola/thread/Mutex.h"
+#include "plugins/usbdmx/Widget.h"
+
+namespace ola {
+namespace plugin {
+namespace usbdmx {
+
+class ShowJockeyDMXU1ThreadedSender;
+
+/**
+ * @brief The ShowJockey-DMX-U1 Widget.
+ *
+ * Stream value to the ShowJockey-DMX-U1 by respecting this packet format:
+ * The first two most significant bytes describe on a uint16_t the index of the
```

Comment:
Sorry, I probably wasn't clear. You've got bytes as follows:
 A B C D

C and D are the first and second channels in the packet respectively.

A and B comprise the two bytes that make up the uint16_t, my question was is A the MSB of the uint16_t or is B? i.e. A << 8 | B or is it B << 8 | A?